### DISCUSSION TITLE
Is the CarlkinKit requirement only for CarPlay or for AndroidAuto as well?

### DISCUSSION THREAD

**Discussion by mathewparet** [tesla-android/issue-tracker#245] [Q&A] [answered]
I've noticed many places where CarPlay is sometimes used to only refer Apple CarPlay and sometimes used for both Apple CarPlay and Android Auto.

When you said CarLinkIt is required for CarPlay do you mean it is required only if I have an iPhone? I have an Android Phone (and no iPhone), do I need CarLinkIt as well for running Android Auto and waze using this project?

**Comment by mikegapinski** [comment, marked as answer]
Carlinkit is required for wireless CarPlay. 

You can use the headunit reloaded app from the Play Store to act as a Wireless Android auto receiver. In this case your phone connects to the Tesla Android Wi-Fi network and uses it for wireless android auto. Make sure to disable the audio sink in the app settings to ensure bluetooth is used for audio between your Android phone and the car.

If you want only Android Auto you can use TeslaAA, it is not perfect but if you don't need all other Tesla Android features it is a simpler solution.

If you don't want Android Auto you'll be able to run standalone navigation apps in Tesla Android soon. I am integrating the GPS from the MCU; navigation sounds are already handled(Tesla Android pumps audio out to the browser without having to pair the pi with the car over BT, this is used for calls).
